plugins/reference-assets/core/src/assets/choice/choices-entry/transform.ts

```diff
@@ -0,0 +1,17 @@
+import type { ChoicesEntry, TransformedChoicesEntry } from './types';
+
+/**
+ * Choice asset transform
+ */
+export const choicesEntryTransform: (entry: ChoicesEntry) => TransformedChoicesEntry = (
```

Comment:
I don't think this is needed, and can be handled in the main transform, to ensure proper scope to capture the `binding` for adding the `select` function, and `selected` property